<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BattleDetail</class>
 <widget class="QDialog" name="BattleDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>520</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="firstPersonName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="secondPersonName">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>10</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableView" name="tableFirstRound">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>1141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableSecondRound">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>1141</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableThirdRound">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>1141</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Подсчитать кол-во победных сходов</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstQntWins">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="secondQntWins">
   <property name="geometry">
    <rect>
     <x>946</x>
     <y>40</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>470</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Завершить бой</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstPersonPoints">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="secondPersonPoints">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>70</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
